File: PASCAL/Cap07/c07ex03.pas
program c07ex03;

var
  A, B: Integer;

procedure TROCA;
var
  X: Integer;
begin
  X := A;
  A := B;
  B := X;
end;

begin
  Write('Entre um valor para a variavel <A>: ');
  ReadLn(A);
    
  Write('Entre um valor para a variavel <B>: ');
  ReadLn(B);

  TROCA;

  WriteLn;

  WriteLn('Variavel <A> com valor ', A);
  WriteLn('Variavel <B> com valor ', B);

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: CSHARP/Cap07/c07ex03.cs
// c07ex03.cs

using System;
using static System.Console;

class Program
{
    static int A, B;

    static void Troca()
    {
        int X = A;
        A = B;
        B = X;
    }

    public static void Main(string[] args)
    {
        Write("Entre um valor para a variavel <A>: ");
        A = int.Parse(ReadLine());

        Write("Entre um valor para a variavel <B>: ");
        B = int.Parse(ReadLine());

        Troca();

        WriteLine();
        WriteLine("Variavel <A> com valor " + A);
        WriteLine("Variavel <B> com valor " + B);

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}
